test/e2e/main-e2e.test.js

```diff
@@ -60,4 +61,13 @@ describe('application launch', function () {
     await client.pause(500);
     await client.getWindowCount().should.eventually.equal(initialWindowCount + 1);
   });
+
+  it('check that WebDriverAgent private headers folder (regression test for https://github.com/appium/appium-desktop/issues/417)', async function () {
+    // NOTE: This isn't really an "e2e" test, but the test has to be written here because the /release 
+    // folder needs to be built in order to run the test
+    if (platform === 'darwin') {
```

Comment:
I would rather the logic be to skip the test if it is on a system that is not Mac.
```js
if (platform === 'darwin') {
  return this.skip();
}
// now do the test
```